lte/gateway/c/core/oai/tasks/nas/emm/EmmStatusHdl.c

```diff
@@ -115,6 +114,11 @@ int emm_proc_status(mme_ue_s1ap_id_t ue_id, emm_cause_t emm_cause) {
     if (ctx) {
       sctx = &ctx->_security;
     }
+    OAILOG_INFO_UE(
```

Comment:
Check if ctx is not-null here.

Reply:
ctx can't be null here since ue_mm_context is not null. (  emm_context_t emm_context; is part of struct mme_ue_context_t)